python/solid_dmft/postprocessing/plot_correlated_bands.py

```diff
@@ -428,41 +542,55 @@ def plot_bands(fig, ax, alatt_k_w, tb_data, freq_dict, n_orb, dft_mu, tb=True, a
         for band in range(n_orb):
             if not proj_on_orb[0] is not None:
                 color = eval('cm.'+plot_dict['colorscheme_bands'])(1.0)
-                ax.plot(tb_data['k_mesh'], eps_nuk[band].real - dft_mu, c=color, label=r'tight-binding', zorder=1.)
+                ax.plot(tb_data['k_mesh'], eps_nuk[band, band].real - tb_data['efermi'], c=color, label=r'tight-binding', zorder=1.)
             else:
                 total_proj = np.zeros(np.shape(evec_nuk[0, band]))
                 for orb in proj_on_orb:
                     total_proj += np.real(evec_nuk[orb, band] * evec_nuk[orb, band].conjugate())
                 color = eval('cm.'+plot_dict['colorscheme_bands'])(total_proj)
-                ax.scatter(tb_data['k_mesh'], eps_nuk[band].real - dft_mu, c=color, s=1, label=r'tight-binding', zorder=1.)
+                ax.scatter(tb_data['k_mesh'], eps_nuk[band, band].real - tb_data['efermi'], c=color, s=1, label=r'tight-binding', zorder=1.)
 
     _setup_plot_bands(ax, tb_data['special_k'], tb_data['k_points_labels'], freq_dict)
 
+
 def plot_kslice(fig, ax, alatt_k_w, tb_data, freq_dict, n_orb, tb_dict, tb=True, alatt=False, quarter=0, **plot_dict):
 
     proj_on_orb, quarter = check_and_convert_plotting(**plot_dict, quarter=quarter)
 
-    sign = [1,-1]
-    quarters = np.array([sign,sign])
+    sign = [1, -1]
+    quarters = np.array([sign, sign])
     four_quarters = list(itertools.product(*quarters))
     used_quarters = [four_quarters[x] for x in quarter]
 
+    if 'vmax' in plot_dict:
+        vmax = plot_dict['vmax']
+    else:
+        vmax = np.max(alatt_k_w)
+    if 'vmin' in plot_dict:
+        vmin = plot_dict['vmin']
+    else:
+        vmin = 0.0
```

Comment:
Could we shorten this to
`vmax = plot_dict['vmax'] if 'vmax' in plot_dict else np.max(alatt_k_w)`
and
`vmin = plot_dict['vmin'] if 'vmin' in plot_dict else 0.0`
?